src/graphrag_toolkit/indexing/load/bedrock_knowledge_base.py

```
@@ -168,53 +191,59 @@ def _with_page_number(self, metadata, page_number):
-                    metadata = json.loads(kb_chunk['AMAZON_BEDROCK_METADATA'])
-                    source = metadata['source']
-                    
-                    chunk = TextNode()
-                    
-                    
-                    chunk.text = kb_chunk['AMAZON_BEDROCK_TEXT']
-                    chunk.metadata = metadata
-                    chunk.metadata['bedrock_id'] = bedrock_id
-                    
-                    (publish_doc, doc) = self._get_source_doc(source_docs_dir, source)
+        with TempDir(job_dir) as j, TempDir(bedrock_dir) as k, TempDir(llama_index_dir) as s:
```

Ruff (0.8.2)

Local variable `j` is assigned to but never used

Remove assignment to unused variable `j`

(F841)